validator/src/main.rs

```diff
@@ -1287,6 +1287,14 @@ pub fn main() {
                 .default_value(&default_rpc_threads)
                 .help("Number of threads to use for servicing RPC requests"),
         )
+        .arg(
+            Arg::with_name("rpc_timeout")
```

Comment:
I'm ok calling this `--rpc-bigtable-timeout`, and then plumbing `rpc_bigtable_timeout` throughout this PR, that's really what this arg is after all

Reply:
done at https://github.com/solana-labs/solana/pull/14657/commits/2b73a6c1ce44ba1511c66fcd6be431d6999f46f4